hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OmTableInsightTask.java

```diff
@@ -108,6 +108,9 @@ public void init() {
    */
   @Override
   public TaskResult reprocess(OMMetadataManager omMetadataManager) {
+    LOG.info("Starting RocksDB Reprocess for {}", getTaskName());
```

Comment:
I think metrics are added for all task, do we need same in info log also here at this granularity?